src/TextInputWidget.cc

```diff
@@ -101,6 +105,39 @@ FilteredTextEdit::keyPressEvent(QKeyEvent *event)
         }
 }
 
+bool
+FilteredTextEdit::canInsertFromMimeData(const QMimeData *source) const
+{
+        return (source->hasImage() || source->hasText() ||
+                QTextEdit::canInsertFromMimeData(source));
+}
+
+void
+FilteredTextEdit::insertFromMimeData(const QMimeData *source)
+{
+        if (source->hasImage()) {
+                QPixmap const img = qvariant_cast<QPixmap>(source->imageData());
```

Comment:
[`QMimeData::imageData`](https://doc.qt.io/qt-5/qmimedata.html#imageData) is documented as returning a `QVariant` containing a `QImage`. Why is a `QPixmap` used here instead?